backend/src/test/java/com/google/sps/FindMatchQueryTest.java

```diff
@@ -12,10 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps;
+package test.java.com.google.sps;
 
 import static com.google.common.truth.Truth.assertThat;
 
+import com.google.sps.FindMatchQuery;
```

Comment:
Why do we need to fix this import in this PR?

Reply:
The naming was wrongly changed, I was just correcting it